docs/develop/gpu.md

```diff
@@ -1,9 +1,18 @@
-# GPU Support
+# GPU support
+
+{% hint style="warning" %}
 
 GPU support is a newer feature and limited to certain hardware, but we use it in production at Baseten.
 
-GPU support is independent from what hardware your model was trained on. Rather, it asks if your model requires a GPU to run predictions.
+{% endhint %}
+
+Large machine learning models are generally trained on a CUDA-based GPU. Once trained, some can return predictions on a standard CPU, but others require access to GPU hardware. For these cases, Truss creates a model serving environment that gives GPU access.
+
+To enable GPU support, go to your Truss' `config.yaml` and set:
 
-Essentially, to enable GPU support, just set `gpu: true` in the config file and the Docker image updates to include CUDA. This Docker image will run on many, but not all, devices that don't support CUDA with a substantial reduction in performance.
+```
+resources:
+  use_gpu: true
+```
 
-Non-CUDA GPUs are not yet supported.
+This will ensure the Docker image created to serve your Truss includes CUDA and accesses the system's GPU, if available. This Docker image will run on many, but not all, devices that don't support CUDA with a substantial reduction in performance.
```

Comment:
The second sentence seems a bit confusing, I wonder if it can be reworded to read more simply.